airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -48,7 +48,7 @@ export const JobItem: React.FC<JobItemProps> = ({ job }) => {
   const didSucceed = didJobSucceed(job);
 
   const onExpand = () => {
-    setIsOpen(!isOpen);
+    setIsOpen((prevIsOpen) => !prevIsOpen);
```

Comment:
Just updating this to use the best practice of using the arrow function for a change to a state value that depends on itself